docs/quick-start-guide.md

```diff
@@ -134,10 +134,6 @@ discouraged either). Meaning it might work but in the case it doesn't you would
 have to provide your own patches to fix problems (the project maintainer and
 community are glad to assist where needed).
 
-There are currently some issues that make `mgmt` less suitable to run for
```

Comment:
Why kill this? No systemd support or equivalent on osx...

Reply:
True, I just couldn't find any issue to reference :)